tensortrade/agents/dqn_agent.py

```diff
@@ -168,6 +171,9 @@ def train(self,
             if save_path and (is_checkpoint or episode == n_episodes - 1):
                 self.save(save_path, episode=episode)
 
+            self.env.render(episode + 1)
+            # self.env.viewer.save(f'episode_{episode}_step_{steps_done}')
```

Comment:
Remove commented line

Reply:
Done in daceea2. This was intentionally left to uncomment when the automatic chart file saving is tested.

Reply:
Ahh, I see. Perhaps a new PR can be opened when this is ready.